ddtrace/contrib/aiopg/__init__.py

```diff
@@ -1,7 +1,5 @@
-"""Instrument aiopg to report Postgres queries.
-
-``patch`` will automatically patch your aiopg connection to make it work.
-::
+"""Instrument aiopg to report Postgres queries. ``patch_all()`` will automatically
```

Comment:
Not sure we need to refer to `patch_all`here?
But good to be more precise about we generate (one span per PG query).